vscode/src/testController.ts

```diff
@@ -184,8 +184,18 @@ export class TestController {
       this.terminal = this.getTerminal();
     }
 
+    const activeEditor = vscode.window.activeTextEditor;
     this.terminal.show();
     this.terminal.sendText(command);
+    if (activeEditor) {
+      setTimeout(() => {
```

Comment:
Without a timeout here it fails. I'm unsure what would be a better way.

Reply:
Updated

Reply:
Yes, but the commits are different for some reason. That branch is on 2e0331d while this PR is on 14026c6.